certbot-apache/certbot_apache/_internal/configurator.py

```diff
@@ -162,10 +162,10 @@ def pick_apache_config(self, warn_on_no_mod_ssl: bool = True) -> str:
         """
         # Disabling TLS session tickets is supported by Apache 2.4.11+ and OpenSSL 1.0.2l+.
         # So for old versions of Apache we pick a configuration without this option.
-        min_openssl_version = util.parse_loose_version('1.0.2l')
+        min_openssl_version = util.LooseVersion('1.0.2l')
         openssl_version = self.openssl_version(warn_on_no_mod_ssl)
         if self.version < (2, 4, 11) or not openssl_version or \
-            util.parse_loose_version(openssl_version) < min_openssl_version:
+            util.LooseVersion(openssl_version).try_risky_less(min_openssl_version):
```

Comment:
I think that with the approach taken in this PR each use of the class should probably handle the case where the objects are incomparable unless we're either certain that won't happen or are OK with a `TypeError` being raised out of the function using `LooseVersion`.

If you want to try and make judgement calls about what we should do in each case, feel free, but otherwise maybe don't switch (all) callers over to the new comparison function and leave `parse_loose_version` undeprecated for now. Switching over the rest and deprecating the function could be saved for future work.